cmd/config/fixtures/config/config.yaml

```diff
@@ -4,6 +4,9 @@ cloud-init-paths:
 - "some/alternate/path"
 
 install:
+  partitions:
+    efi:
+      size: 512
```

Comment:
How/where do we document the size unit ? (I'd assume it's in megabytes ;-) )

Reply:
Yes everything is in megabytes, it is already added as a comment in the example config file and also in elemental-toolkit docs.